app/main.py

```diff
@@ -1 +1,44 @@
-# write your code here
+class Animal:
+
+    alive = []
+
+    def __init__(self, name: str,
+                 health: int = 100,
+                 hidden: bool = False) -> None:
+
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+
+        if self.health > 0:
+            return Animal.alive.append(self)
```

Comment:
The `append` method returns `None`, so there is no need to use `return` here. Simply use `Animal.alive.append(self)` without `return`.